### PR TITLE
Scratch/cardbox

### PR DESCRIPTION
**Pull request by aditya9940**
CardBox component
Data.json
SVG files
added some data in  page.tsx for reference

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+import CardBox from "@component/cardBox";
+import data from '../data.json'
 export default function Home() {
   return (
     <div className={"container"}>
-      <Hero introduction={["Internet can be a dangerous place,", "Do you sometimes worry"]} content={["About the online scams?", "We hear these worries a lot from our clients, parents, teenagers, and friends outside IT industry, we could go on and on. Security is difficult and people from all layers of security struggle with these questions."]}/>     
-      <Shield top={95} left={1300}/>
+      <Hero introduction={["Internet can be a dangerous place,", "Do you sometimes worry"]} content={["About the online scams?", "We hear these worries a lot from our clients, parents, teenagers, and friends outside IT industry, we could go on and on. Security is difficult and people from all layers of security struggle with these questions."]} />
+      <Shield top={95} left={1300} />
+      <div style={{ padding: "160px", transform: "translateY(-50%)" }}>
```
Comment (arijeet-mukherjee) [inline src/app/page.tsx:11]: we can use calc method for padding here, 160px padding looks good for 1920px for smaller end device we can calculate

File: src/component/cardBox/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            setCardGradient({ background: props.iconPosition == "left" ? "linear-gradient(270.59deg, #FFFFFF 35.89%, #C7D5E0 79.41%)" : "linear-gradient(108.31deg, #FFFFFF 34.81%, #C7D5E0 78.11%)", gridRow: 1 });
+        }
+    }, [])
+
+    const { image, title, description, iconPosition, buttonText } = props;
+    const imageWidth = 'calc((100vw / 1920) * 329.61'
+    const imageHeight = 'calc((100vw / 1920) * 307.29'
+    return (
+        <div className={styles["cardbox-card"]} style={cardGradient} >
+            <div className={styles["cardbox-content"]} style={cardContentStyle} >
```
Comment (arijeet-mukherjee) [inline src/component/cardBox/index.tsx:63]: we need an extra left padding of 115px if iconposition is right.

File: src/app/page.tsx (hunk 1 above)
Comment (aditya9940) [reply src/app/page.tsx:11]: we'll take care this in integration US

File: src/component/cardBox/index.tsx (hunk 2 above)
Comment (aditya9940) [reply src/component/cardBox/index.tsx:64]: resolved

**Review by arijeet-mukherjee — APPROVED** [review]
LGTM

**Pull request merged**